src/Symfony/Contracts/Cache/CacheInterface.php

```diff
@@ -13,6 +13,7 @@
 
 use Psr\Cache\CacheItemInterface;
 use Psr\Cache\InvalidArgumentException;
+use Symfony\Component\Cache\CacheItem;
```

Comment:
referencing the component in the contracts breaks the package principles
the annotation should use `CacheItemInterface|ItemInterface`

Reply:
should it be a union or an intersection though ?

Reply:
it's a union: ItemInterface already extends CacheItemInterface, accepting a CacheItemInterface is desired for PSR-6 compat, and ItemInterface for more advanced items.

Reply:
Done